components/FeatureHighlight/src/MDCFeatureHighlightViewController.h

```diff
@@ -92,47 +94,9 @@ typedef void (^MDCFeatureHighlightCompletion)(BOOL accepted);
 @property(nonatomic, copy, nullable) NSString *bodyText;
 
 /**
- Sets the color to be used for the outer highlight. Defaults to blue with an alpha of
- kMDCFeatureHighlightOuterHighlightAlpha.
-
- Alpha should be set to kMDCFeatureHighlightOuterHighlightAlpha.
- */
-@property(nonatomic, strong, null_resettable) UIColor *outerHighlightColor;
```

Comment:
there are internal clients that are using this and the other properties. We should mark them deprecated instead of going strait to breaking changes.